blueocean-core-js/src/js/security.js

```diff
@@ -30,7 +30,7 @@ function isSecurityEnabled() {
 }
 
 function isAnonymousUser() {
-    return config.getInitialUser() === 'anonymous';
+    return config.getInitialUser().id === 'anonymous';
```

Comment:
Yeah, was just getting it working first. Ideally, that `User` class that's in personalization should really be in core-js and then we could use that with guarded accessor methods etc